sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/EnvironmentVariableConstants.cs

```diff
@@ -56,6 +56,16 @@ internal static class EnvironmentVariableConstants
         /// </summary>
         public const string WEBSITE_SITE_NAME = "WEBSITE_SITE_NAME";
 
+        /// <summary>
+        /// INTERNAL ONLY. Used by Statsbeat to get the Kubernetes Service Host.
+        /// </summary>
+        public const string KUBERNETES_SERVICE_HOST = "KUBERNETES_SERVICE_HOST";
```

Comment:
this is available for any kubernetes, including AKS. 
it's incorrect to send rpId = aks when it's kubernetes. it's not possible to collect denominator for kubernetes at all.
i think for now we would send if AKS_ARM_NAMESPACE_ID when it has value (auto-instrumentation). 
PM will figure out the manual instrumentation numerator by working with AKS directly. We can follow up with them to find out.

Reply:
Are you asking to remove this string _resourceProvider = "aks";?

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/39663#discussion_r1379122997